akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -2921,18 +2921,18 @@ trait FlowOps[+Out, +Mat] {
    */
   def mergePreferred[U >: Out, M](
       that: Graph[SourceShape[U], M],
-      priority: Boolean,
+      preferred: Boolean,
```

Comment:
Changing a parameter name breaks source compatibility in Scala (since it could be used as a named parameter), but I think that you can use `@deprecatedName("priority", "2.6.19") preferred: ...` to still allow the current name but with a deprecation warning and keep compatibility.